packages/matter-node.js/src/matter/common/DeviceTypes.ts

```diff
@@ -3,49 +3,221 @@
+            //MatterClusters.WifiNetworkDiagnosticsCluster.id, // optional if WiFi
+            //MatterClusters.ThreadNetworkDiagnosticsCluster.id, // optional if Thread
+        ],
+        requiredClientClusters: [],
+        optionalClientClusters: [],
+    },
 
     /**
      * This represents a Power Source Node for devices.
      * @see {@link MatterDeviceLibrarySpecificationV1_0} § 2.2
      */
-    POWER_SOURCE: { name: "MA-powersource", code: 0x011 },
+    POWER_SOURCE: {
+        name: "MA-powersource",
+        code: 0x011,
+        class: DeviceClasses.Node, // ???
+        revision: 1,
+        requiredServerClusters: [
+            MatterClusters.PowerSourceCluster.id
+        ],
+        optionalServerClusters: [],
+        requiredClientClusters: [],
+        optionalClientClusters: [],
+    },
 
     /**
      * An OTA Requestor is a device that is capable of receiving an OTA software
      * update.
      * @see {@link MatterDeviceLibrarySpecificationV1_0} § 2.3
      */
-    OTA_REQUESTOR: { name: "MA-otarequestor", code: 0x012 },
+    OTA_REQUESTOR: {
+        name: "MA-otarequestor",
+        code: 0x012,
+        class: DeviceClasses.Node, // ???
+        revision: 1,
+        requiredServerClusters: [
+            //OtaSoftwareUpdateRequestorCluster.id
+        ],
+        optionalServerClusters: [],
+        requiredClientClusters: [
+            //OtaSoftwareUpdateProviderCluster.id
+        ],
+        optionalClientClusters: [],
+    },
 
     /**
      * An OTA Provider is a node that is capable of providing an OTA software
      * update to other nodes on the same fabric.
      * @see {@link MatterDeviceLibrarySpecificationV1_0} § 2.4
      */
-    OTA_PROVIDER: { name: "MA-otaprovider", code: 0x0014 },
+    OTA_PROVIDER: {
+        name: "MA-otaprovider",
+        code: 0x0014,
+        class: DeviceClasses.Node, // ???
+        revision: 1,
+        requiredServerClusters: [
+            //OtaSoftwareUpdateProviderCluster.id
+        ],
+        optionalServerClusters: [],
+        requiredClientClusters: [],
+        optionalClientClusters: [
+            //OtaSoftwareUpdateRequestorCluster.id
+        ],
+    },
 
     /**
      * This represents a Aggregator Node.
      * @see {@link MatterDeviceLibrarySpecificationV1_0} § 2.5
      */
-    AGGREGATOR: { name: "MA-aggregator", code: 0x000e },
+    AGGREGATOR: {
+        name: "MA-aggregator",
+        code: 0x000e,
+        class: DeviceClasses.Dynamic, // ???
+        revision: 1,
+        requiredServerClusters: [],
+        optionalServerClusters: [
+            //MatterClusters.ActionsCluster.id
+        ],
+        requiredClientClusters: [],
+        optionalClientClusters: [],
+    },
+
+    /**
+     * This represents a Bridged Node to identify a bridged device without Power Source information.
+     * @see {@link MatterDeviceLibrarySpecificationV1_0} § 2.6
+     */
+    BRIDGED_DEVICE: {
```

Comment:
Should this and other references be "BRIDGED_NODE" as in standard? i.e. node instead of device

Reply:
valid, renamed